indico/modules/events/models/persons.py

```diff
@@ -244,7 +262,7 @@ def merge_person_info(self, other):
             if existing_event_link is None:
                 event_link.person = self
             else:
-                db.session.delete(event_link)
+                other.event_links.remove(event_link)
```

Comment:
Yeah. Anyway, what I meant is... what's the advantage of having `EventPersonLink` vis a vis a column in `EventPerson` telling us their role?